depends/packages/openssl.mk

```diff
@@ -57,6 +57,7 @@ $(package)_config_opts_riscv64_linux=linux-generic64
 $(package)_config_opts_x86_64_darwin=darwin64-x86_64-cc
 $(package)_config_opts_x86_64_mingw32=mingw64
 $(package)_config_opts_i686_mingw32=mingw
+$(package)_config_opts_android=linux-generic64
```

Comment:
What if it's being built for ARM32, x86, or some other platform entirely?

Reply:
Google has deprecated both 32-bit apps and GCC in favour of Clang. I tried to use OpenSSL's `android` target instead of `linux-generic` but it assumes GCC and then `Config` script's `-mandroid` flag [breaks](https://github.com/openssl/openssl/issues/1531) Clang.

Reply:
So this :- https://github.com/openssl/openssl/pull/2229 would not work ?

Reply:
Actualy, i think this does it for both https://github.com/ruslo/hunter/pull/814/files. Is it worth creating a patch ?

Reply:
Google has not deprecated 32bit apps afaik. Link? It has deprecated gcc in favour of clang that's correct. Most emulators images are still 32 bit AFAIK.

Reply:
If you are referring to https://android-developers.googleblog.com/2019/01/get-your-apps-ready-for-64-bit.html they want people to support 64 but they explicitly state that 32 bit support is here to stay.

Reply:
My bad, you are right, they haven't deprecated 32-bit. But let's continue this discussion in the main comments.